scripts/Muon/GUI/ElementalAnalysis2/grouping_widget/ea_grouping_table_widget_presenter.py

```diff
@@ -176,6 +177,24 @@ def to_analyse_data_checkbox_changed(self, state, group_name):
         self.selected_group_changed_notifier.notify_subscribers(group_info)
 
     def plot_default_case(self):
+        """
+            Detector 3 should be plotted by default and if not present Detecor 1 should be plotted, if neither is
```

Comment:
why detector 3, then 1, 4 and then 2?

Reply:
It is because 3 and 1 are the most useful due to the positioning and detector 3 and 4 is preferential used due to being  low energy detectors with higher resolution.